docs/conf.py

```diff
@@ -41,16 +40,9 @@
 # The suffix(es) of source filenames.
 # You can specify multiple suffix as a list of string:
 # source_suffix = ['.rst', '.md']
-# Markdown support
-
-from recommonmark.parser import CommonMarkParser
```

Comment:
@bast @ilfreddy Why remove the possibility to write in Markdown? ReadTheDocs can handle it, given the appropriate `reqruirements.txt`. For a local build one could get the packages rather easily. Either globally:
```Python
pip install -r docs/requirements.txt
```
or (even better) locally in a [virtual environment] using the same command.

[virtual environment]: http://docs.python-guide.org/en/latest/dev/virtualenvs/